ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -227,13 +235,23 @@ class AdminWorkspaceImpl extends React.Component<UrlParamsProps, State> {
             )}
           </div>
           <h3>Collaborators</h3>
-          <div className='collaborators' style={{marginTop: '1rem'}}>
-            {collaborators.map((userRole, i) =>
-              <div key={i}>
-                {userRole.email + ': ' + userRole.role}
-              </div>
-            )}
-          </div>
+          <FlexRow className='collaborators' style={{marginTop: '1rem'}}>
+            <FlexColumn>
+              {collaborators.map((userRole, i) =>
+                <div key={i}>
+                  {userRole.email + ': ' + userRole.role}
```

Comment:
We may also want to show the disabled status of the users here? Just TOL.

Reply:
How about I turn the collaborators list into a table and include the disabled status there?

Reply:
+1 I also was toying with adding an indication of which one is running the cluster (maybe at the bottom), their full name, contact email, and institution (for product and security) and their database `user_id` (for DB fixes and looking up in the action audit log). I can do the backend pieces under RW-4825.